tests/ui/pages/discopane.py

```diff
@@ -0,0 +1,44 @@
+from pypom import Page
+from selenium.webdriver.common.by import By
+from selenium import webdriver
+import pytest
+
+
+class DiscoveryPane(Page):
+
+    _discopane_content_locator = (By.CSS_SELECTOR, '.disco-pane')
+    _play_video_locator = (By.CSS_SELECTOR, '.play-video')
+    _uninstalled_toggles = (By.CSS_SELECTOR, '.switch.uninstalled')
+    _close_video_link = (By.CSS_SELECTOR, '.close-video')
+    _see_more_addons_locator = (By.CSS_SELECTOR, '.amo-link')
+    _search_box_locator = (By.ID, 'lst-ib')
+
+    def open_addons(self):
+        elem = self.find_element(*self._search_box_locator)
+        elem.send_keys(Keys.COMMAND + Keys.SHIFT + 'a')
```

Comment:
This isn't used yet, and when we introduce it I don't think it would be as a method in the `DiscoveryPane` object. I think we'd have a higher level helper, or it would be in the `AboutAddons.open` method. For now, let's remove it.